www/includes/easyparliament/templates/html/people/index.php

```diff
@@ -159,11 +160,17 @@
                     <a href="#Y">Y</a>
                     <a href="#Z">Z</a>
                 </div>
+                <?php } ?>
 
                 <div class="people-list">
-                <?php $current_initial = ''; ?>
-                <?php foreach ( $data as $person ) {
-                    $initial = substr( strtoupper($person['family_name']), 0, 1);
+                <?php
```

Comment:
Could this not be output if order is party too? Given it's not used (and is outputting multiple times)